2018-06-28 09:31:21.002 27732 DEBUG nova.api.openstack.wsgi [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:31:21.024 27732 DEBUG nova.compute.api [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Searching by: {u'changes-since': datetime.datetime(2018, 6, 28, 7, 26, 43, 363244, tzinfo=<FixedOffset u'+00:00' datetime.timedelta(0)>)} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:31:21.053 27732 DEBUG oslo_concurrency.lockutils [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:31:21.054 27732 DEBUG oslo_concurrency.lockutils [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:31:21.169 27732 DEBUG nova.compute.api [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:31:21.170 27732 DEBUG nova.compute.api [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:31:21.172 27732 DEBUG oslo_concurrency.lockutils [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:31:21.173 27732 DEBUG oslo_concurrency.lockutils [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:31:21.318 27732 DEBUG nova.policy [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'0508a2373b814911b1d6723f2f75b8cd', 'roles': [u'ResellerAdmin', u'admin'], 'user_domain_id': u'default', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'910acf3b84b64f358e48f506530f7cb1', 'project_domain_id': u'default'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:31:21.332 27732 INFO nova.osapi_compute.wsgi.server [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] 10.0.20.25 "GET /v2.1/910acf3b84b64f358e48f506530f7cb1/servers/detail?all_tenants=True&changes-since=2018-06-28T07%3A26%3A43.363244%2B00%3A00 HTTP/1.1" status: 200 len: 413 time: 0.3454540
2018-06-28 09:31:44.901 27733 DEBUG nova.osapi_compute.wsgi.server [req-b9c00747-4a48-4dbf-a775-22bc8a638fea 82758ee9dbed4bdf919e32bc6135cc36 f7db0ec7eef2499fb14973cea9da1187 - default default] (27733) accepted ('10.0.20.25', 56858) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:31:45.375 27733 DEBUG nova.api.openstack.wsgi [req-2a95f67a-73a3-4013-9f1e-5b16fb34de1d 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Azione: 'create', metodo chiamata: <function version_select at 0x7f4f4c17a140>, corpo: {"keypair": {"public_key": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC/IOEMpVJk1CXrnyZgwnwPOqbgHy6HczhiPTWfesqrMyQfrIS7xLL3meeOFJk1ARGokE54r3UErmuvQhXpGX+8JazNNhCDUT26viaPj0Q8zTEOporU9M1IoOeK3IXafOD6bdsSoknPnnyibExkMBYfYqxsGFIjI6TXMVqlBwtHVmUk+8rZBj0AzWuZ5blMn8jKOAuCzz7UZ01LCP7ybyoq+t5U8nnwx+ihN5zPNu3T03kesPpRu7OwduHH+RskGTgbECuIGzYPLIe+lJNKkL464ZC2qmgDbOmfwTYeJcAXgWfU8lzKyaraiWgDCspR4MTgLjoTXcVoMuRkf/G0IJFj root@localhost.localdomain\n", "name": "tempest-keypair-1530171027"}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-28 09:31:45.488 27733 DEBUG nova.quota [req-2a95f67a-73a3-4013-9f1e-5b16fb34de1d 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Getting quotas for project 51224f7169ad4f1baed59aaad0b51497. Resources: set(['key_pairs']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-28 09:31:45.499 27733 DEBUG nova.quota [req-2a95f67a-73a3-4013-9f1e-5b16fb34de1d 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Getting quotas for user 9e5ee4e333bc4c978c23ee6dbc8bbc5d and project 51224f7169ad4f1baed59aaad0b51497. Resources: set(['key_pairs']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-28 09:31:45.630 27733 INFO nova.osapi_compute.wsgi.server [req-2a95f67a-73a3-4013-9f1e-5b16fb34de1d 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "POST /v2.1/51224f7169ad4f1baed59aaad0b51497/os-keypairs HTTP/1.1" status: 200 len: 997 time: 0.7252870
2018-06-28 09:31:52.512 27729 DEBUG nova.osapi_compute.wsgi.server [req-ba70d1f1-2b7e-48a0-9964-ff4246d73267 - - - - -] (27729) accepted ('10.0.20.25', 56878) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:31:52.523 27729 WARNING keystonemiddleware.auth_token [req-ba70d1f1-2b7e-48a0-9964-ff4246d73267 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-28 09:31:54.272 27729 DEBUG nova.api.openstack.wsgi [req-60cdde7a-f65b-422d-9005-53b8d24850e2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<function version_select at 0x7f4f4c1a20c8>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:31:54.327 27729 DEBUG oslo_db.sqlalchemy.engines [req-60cdde7a-f65b-422d-9005-53b8d24850e2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:31:54.340 27729 WARNING oslo_db.sqlalchemy.utils [req-60cdde7a-f65b-422d-9005-53b8d24850e2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Unique keys not in sort_keys. The sorting order may be unstable.
2018-06-28 09:31:54.359 27729 WARNING oslo_db.sqlalchemy.utils [req-60cdde7a-f65b-422d-9005-53b8d24850e2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Unique keys not in sort_keys. The sorting order may be unstable.
2018-06-28 09:31:54.377 27729 INFO nova.osapi_compute.wsgi.server [req-60cdde7a-f65b-422d-9005-53b8d24850e2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/os-keypairs HTTP/1.1" status: 200 len: 966 time: 1.8591230
2018-06-28 09:33:08.047 27733 DEBUG nova.api.openstack.wsgi [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:33:08.062 27733 DEBUG nova.compute.api [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Searching by: {u'changes-since': datetime.datetime(2018, 6, 28, 7, 28, 10, 542477, tzinfo=<FixedOffset u'+00:00' datetime.timedelta(0)>)} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:33:08.088 27733 DEBUG oslo_concurrency.lockutils [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:33:08.089 27733 DEBUG oslo_concurrency.lockutils [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:33:08.197 27733 DEBUG nova.compute.api [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:33:08.198 27733 DEBUG nova.compute.api [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:33:08.200 27733 DEBUG oslo_concurrency.lockutils [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:33:08.200 27733 DEBUG oslo_concurrency.lockutils [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:33:08.356 27733 DEBUG nova.policy [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'0508a2373b814911b1d6723f2f75b8cd', 'roles': [u'ResellerAdmin', u'admin'], 'user_domain_id': u'default', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'910acf3b84b64f358e48f506530f7cb1', 'project_domain_id': u'default'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:33:08.370 27733 INFO nova.osapi_compute.wsgi.server [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] 10.0.20.25 "GET /v2.1/910acf3b84b64f358e48f506530f7cb1/servers/detail?all_tenants=True&changes-since=2018-06-28T07%3A28%3A10.542477%2B00%3A00 HTTP/1.1" status: 200 len: 413 time: 0.3396890
2018-06-28 09:34:14.814 27728 DEBUG nova.osapi_compute.wsgi.server [req-ba70d1f1-2b7e-48a0-9964-ff4246d73267 - - - - -] (27728) accepted ('10.0.20.25', 59368) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:34:14.845 27728 WARNING keystonemiddleware.auth_token [req-ba70d1f1-2b7e-48a0-9964-ff4246d73267 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-28 09:34:16.942 27728 DEBUG nova.api.openstack.wsgi [req-83586cb1-c909-479a-b6c9-3c8a2c008d87 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method FlavorsController.show of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7f4f4cbd98d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:16.994 27728 DEBUG oslo_db.sqlalchemy.engines [req-83586cb1-c909-479a-b6c9-3c8a2c008d87 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:34:17.101 27728 INFO nova.api.openstack.wsgi [req-83586cb1-c909-479a-b6c9-3c8a2c008d87 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] HTTP exception thrown: Impossibile trovare la tipologia m1.tiny.
2018-06-28 09:34:17.103 27728 DEBUG nova.api.openstack.wsgi [req-83586cb1-c909-479a-b6c9-3c8a2c008d87 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Returning 404 to user: Impossibile trovare la tipologia m1.tiny. __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-28 09:34:17.105 27728 INFO nova.osapi_compute.wsgi.server [req-83586cb1-c909-479a-b6c9-3c8a2c008d87 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/flavors/m1.tiny HTTP/1.1" status: 404 len: 507 time: 2.2733481
2018-06-28 09:34:17.142 27728 DEBUG nova.api.openstack.wsgi [req-cf726d06-9638-4ed6-993b-c42797a5182e 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method FlavorsController.index of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7f4f4cbd90d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:17.221 27728 INFO nova.osapi_compute.wsgi.server [req-cf726d06-9638-4ed6-993b-c42797a5182e 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/flavors HTTP/1.1" status: 200 len: 2106 time: 0.0943549
2018-06-28 09:34:17.257 27728 DEBUG nova.api.openstack.wsgi [req-cfafa476-6367-499a-9178-d85c786f5ee0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method FlavorsController.show of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7f4f4cbd98d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:17.295 27728 INFO nova.osapi_compute.wsgi.server [req-cfafa476-6367-499a-9178-d85c786f5ee0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/flavors/1 HTTP/1.1" status: 200 len: 819 time: 0.0489020
2018-06-28 09:34:20.552 27728 DEBUG nova.api.openstack.wsgi [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Azione: 'create', metodo chiamata: <bound method ServersController.create of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c938ad0>>, corpo: {"server": {"name": "tempest-INSTANCE_SAMPLE-1530171027", "imageRef": "1aeece9b-8a13-4130-9476-499504968ab4", "key_name": "tempest-keypair-1530171027", "flavorRef": "1", "max_count": 1, "min_count": 1, "networks": [{"uuid": "076ee164-c39d-4ac7-8f6d-1cdf41d92deb"}], "security_groups": [{"name": "61234139-da54-4ee7-985a-f97a51586e24"}]}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-28 09:34:20.700 27728 DEBUG oslo_concurrency.lockutils [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:20.703 27728 DEBUG oslo_concurrency.lockutils [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:20.736 27728 DEBUG oslo_db.sqlalchemy.engines [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:34:20.754 27728 DEBUG oslo_concurrency.lockutils [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:20.761 27728 DEBUG oslo_concurrency.lockutils [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.007s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:20.789 27728 DEBUG oslo_db.sqlalchemy.engines [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:34:20.846 27728 WARNING nova.image.glance [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] No protocol specified in for api_server 'http://10.0.20.25:9292', please update [glance] api_servers with fully qualified url including scheme (http / https)
2018-06-28 09:34:22.907 27728 DEBUG nova.network.neutronv2.api [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] validate_networks() for [(u'076ee164-c39d-4ac7-8f6d-1cdf41d92deb', None, None, None)] validate_networks /usr/lib/python2.7/site-packages/nova/network/neutronv2/api.py:1692
2018-06-28 09:34:23.494 27728 DEBUG nova.virt.hardware [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] emulator threads policy constraint: None get_emulator_threads_constraint /usr/lib/python2.7/site-packages/nova/virt/hardware.py:1289
2018-06-28 09:34:23.539 27728 DEBUG nova.quota [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Getting quotas for project 51224f7169ad4f1baed59aaad0b51497. Resources: ['metadata_items'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-28 09:34:23.552 27728 DEBUG nova.quota [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Getting quotas for user 9e5ee4e333bc4c978c23ee6dbc8bbc5d and project 51224f7169ad4f1baed59aaad0b51497. Resources: ['metadata_items'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-28 09:34:23.994 27728 DEBUG nova.quota [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Getting quotas for project 51224f7169ad4f1baed59aaad0b51497. Resources: ['injected_files'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-28 09:34:24.002 27728 DEBUG nova.quota [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Getting quotas for user 9e5ee4e333bc4c978c23ee6dbc8bbc5d and project 51224f7169ad4f1baed59aaad0b51497. Resources: ['injected_files'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-28 09:34:24.040 27728 DEBUG nova.quota [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Getting quotas for project 51224f7169ad4f1baed59aaad0b51497. Resources: ['injected_file_content_bytes', 'injected_file_path_bytes'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-28 09:34:24.049 27728 DEBUG nova.quota [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Getting quotas for user 9e5ee4e333bc4c978c23ee6dbc8bbc5d and project 51224f7169ad4f1baed59aaad0b51497. Resources: ['injected_file_content_bytes', 'injected_file_path_bytes'] _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-28 09:34:24.079 27728 DEBUG nova.context [req-a6a402b2-3b37-4b66-97f8-a151539ddfdf - - - - -] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),207d3421-8e68-4ec6-a61c-47da5c989f5f(default) load_cells /usr/lib/python2.7/site-packages/nova/context.py:508
2018-06-28 09:34:24.081 27728 DEBUG oslo_concurrency.lockutils [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:24.082 27728 DEBUG oslo_concurrency.lockutils [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:24.084 27728 DEBUG oslo_concurrency.lockutils [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:24.085 27728 DEBUG oslo_concurrency.lockutils [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:24.119 27728 DEBUG nova.quota [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Getting quotas for project 51224f7169ad4f1baed59aaad0b51497. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:442
2018-06-28 09:34:24.127 27728 DEBUG nova.quota [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Getting quotas for user 9e5ee4e333bc4c978c23ee6dbc8bbc5d and project 51224f7169ad4f1baed59aaad0b51497. Resources: set(['instances', 'ram', 'cores']) _get_quotas /usr/lib/python2.7/site-packages/nova/quota.py:434
2018-06-28 09:34:24.150 27728 DEBUG nova.compute.api [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Going to run 1 instances... _provision_instances /usr/lib/python2.7/site-packages/nova/compute/api.py:891
2018-06-28 09:34:24.199 27728 DEBUG nova.compute.api [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] block_device_mapping [BlockDeviceMapping(attachment_id=<?>,boot_index=0,connection_info=None,created_at=<?>,delete_on_termination=True,deleted=<?>,deleted_at=<?>,destination_type='local',device_name=None,device_type='disk',disk_bus=None,guest_format=None,id=<?>,image_id='1aeece9b-8a13-4130-9476-499504968ab4',instance=<?>,instance_uuid=<?>,no_device=False,snapshot_id=None,source_type='image',tag=None,updated_at=<?>,volume_id=None,volume_size=None)] _bdm_validate_set_size_and_instance /usr/lib/python2.7/site-packages/nova/compute/api.py:1268
2018-06-28 09:34:24.489 27728 INFO nova.osapi_compute.wsgi.server [req-b860962c-4441-4792-8c8f-68594f586e32 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "POST /v2.1/51224f7169ad4f1baed59aaad0b51497/servers HTTP/1.1" status: 202 len: 989 time: 3.9535789
2018-06-28 09:34:24.526 27728 DEBUG nova.api.openstack.wsgi [req-17dec0a8-448f-4add-9b8f-ce47ba30f29e 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:24.533 27728 DEBUG nova.compute.api [req-17dec0a8-448f-4add-9b8f-ce47ba30f29e 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:34:24.644 27728 DEBUG nova.policy [req-17dec0a8-448f-4add-9b8f-ce47ba30f29e 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:34:24.770 27728 INFO nova.osapi_compute.wsgi.server [req-17dec0a8-448f-4add-9b8f-ce47ba30f29e 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 1857 time: 0.2621970
2018-06-28 09:34:24.844 27728 DEBUG nova.api.openstack.wsgi [req-37f52f09-c560-4e7a-b6c6-5a6155bea4f4 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method FlavorsController.show of <nova.api.openstack.compute.flavors.FlavorsController object at 0x7f4f4cbd98d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:24.885 27728 INFO nova.osapi_compute.wsgi.server [req-37f52f09-c560-4e7a-b6c6-5a6155bea4f4 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/flavors/1 HTTP/1.1" status: 200 len: 819 time: 0.0553041
2018-06-28 09:34:31.993 27729 DEBUG nova.osapi_compute.wsgi.server [req-60cdde7a-f65b-422d-9005-53b8d24850e2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27729) accepted ('10.0.20.25', 59488) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:34:32.586 27729 INFO nova.osapi_compute.wsgi.server [req-4331d7bd-da20-41bd-a7f1-83ad0ad6ad36 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497 HTTP/1.1" status: 404 len: 382 time: 0.5891519
2018-06-28 09:34:32.610 27729 DEBUG nova.api.openstack.wsgi [req-435f9e0e-964d-4fd7-8337-9b918ee8722a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7f4f4cccbd10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:32.614 27729 INFO nova.osapi_compute.wsgi.server [req-435f9e0e-964d-4fd7-8337-9b918ee8722a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0162430
2018-06-28 09:34:33.093 27729 DEBUG nova.api.openstack.wsgi [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:33.105 27729 DEBUG nova.compute.api [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:34:33.150 27729 DEBUG oslo_concurrency.lockutils [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:33.153 27729 DEBUG oslo_concurrency.lockutils [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:33.180 27729 DEBUG oslo_db.sqlalchemy.engines [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:34:33.310 27729 DEBUG nova.compute.api [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),207d3421-8e68-4ec6-a61c-47da5c989f5f(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:236
2018-06-28 09:34:33.311 27729 DEBUG nova.compute.api [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:34:33.312 27729 DEBUG nova.compute.api [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:34:33.314 27729 DEBUG oslo_concurrency.lockutils [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:33.322 27729 DEBUG oslo_concurrency.lockutils [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.008s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:33.349 27729 DEBUG oslo_db.sqlalchemy.engines [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:34:33.486 27729 INFO nova.osapi_compute.wsgi.server [req-fd761d4b-6478-4225-bed6-3daa10a8ce4f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 786 time: 0.4112060
2018-06-28 09:34:33.529 27729 DEBUG nova.api.openstack.wsgi [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:33.536 27729 DEBUG nova.compute.api [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:34:33.573 27729 DEBUG oslo_concurrency.lockutils [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:33.574 27729 DEBUG oslo_concurrency.lockutils [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:34.278 27729 DEBUG oslo_concurrency.lockutils [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:34.280 27729 DEBUG oslo_concurrency.lockutils [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:34.281 27729 DEBUG nova.objects.instance [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:34:34.369 27729 DEBUG nova.policy [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:34:35.621 27729 INFO nova.osapi_compute.wsgi.server [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3860 time: 2.1103010
2018-06-28 09:34:44.894 27729 DEBUG nova.osapi_compute.wsgi.server [req-6628d6b8-ff55-4fe3-a873-76d8879294fe 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27729) accepted ('10.0.20.25', 60574) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:34:45.449 27729 INFO nova.osapi_compute.wsgi.server [req-ae5cefc0-ca5c-4f85-8ffd-d5c684fc8206 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497 HTTP/1.1" status: 404 len: 382 time: 0.5510130
2018-06-28 09:34:45.477 27729 DEBUG nova.api.openstack.wsgi [req-03242230-49c2-4bd1-a25b-cbda2d3b0323 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7f4f4cccbd10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:45.479 27729 INFO nova.osapi_compute.wsgi.server [req-03242230-49c2-4bd1-a25b-cbda2d3b0323 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0165532
2018-06-28 09:34:45.780 27729 DEBUG nova.api.openstack.wsgi [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:45.788 27729 DEBUG nova.compute.api [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:34:45.807 27729 DEBUG oslo_concurrency.lockutils [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:45.808 27729 DEBUG oslo_concurrency.lockutils [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:45.877 27729 DEBUG nova.compute.api [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:34:45.878 27729 DEBUG nova.compute.api [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:34:45.879 27729 DEBUG oslo_concurrency.lockutils [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:45.880 27729 DEBUG oslo_concurrency.lockutils [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:45.949 27729 INFO nova.osapi_compute.wsgi.server [req-71f8f8cc-9143-41a9-b764-99ca675b8c63 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 786 time: 0.1870198
2018-06-28 09:34:45.974 27729 DEBUG nova.api.openstack.wsgi [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:45.979 27729 DEBUG nova.compute.api [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:34:45.995 27729 DEBUG oslo_concurrency.lockutils [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:45.997 27729 DEBUG oslo_concurrency.lockutils [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:46.121 27729 DEBUG oslo_concurrency.lockutils [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:46.122 27729 DEBUG oslo_concurrency.lockutils [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:46.123 27729 DEBUG nova.objects.instance [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:34:46.169 27729 DEBUG nova.policy [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:34:46.905 27729 INFO nova.osapi_compute.wsgi.server [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3860 time: 0.9417701
2018-06-28 09:34:58.394 27729 DEBUG nova.osapi_compute.wsgi.server [req-5c9e22f3-a7d3-4785-9a77-85c7de2c43b2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27729) accepted ('10.0.20.25', 60610) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:34:58.910 27729 INFO nova.osapi_compute.wsgi.server [req-eb5d03e2-5789-4774-a918-92edea67162f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497 HTTP/1.1" status: 404 len: 382 time: 0.5135601
2018-06-28 09:34:58.960 27729 DEBUG nova.api.openstack.wsgi [req-616f36c4-2ba1-42d7-9359-d0d34078a34a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7f4f4cccbd10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:58.964 27729 INFO nova.osapi_compute.wsgi.server [req-616f36c4-2ba1-42d7-9359-d0d34078a34a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0214739
2018-06-28 09:34:59.522 27729 DEBUG nova.api.openstack.wsgi [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:59.535 27729 DEBUG nova.compute.api [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:34:59.560 27729 DEBUG oslo_concurrency.lockutils [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:59.561 27729 DEBUG oslo_concurrency.lockutils [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:59.658 27729 DEBUG nova.compute.api [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:34:59.660 27729 DEBUG nova.compute.api [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:34:59.662 27729 DEBUG oslo_concurrency.lockutils [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:59.663 27729 DEBUG oslo_concurrency.lockutils [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:59.777 27729 INFO nova.osapi_compute.wsgi.server [req-7fc93ee5-7c9b-481b-86d9-1315dfc0ecd3 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 786 time: 0.2760611
2018-06-28 09:34:59.815 27729 DEBUG nova.api.openstack.wsgi [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:34:59.822 27729 DEBUG nova.compute.api [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:34:59.841 27729 DEBUG oslo_concurrency.lockutils [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:59.842 27729 DEBUG oslo_concurrency.lockutils [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:59.975 27729 DEBUG oslo_concurrency.lockutils [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:34:59.977 27729 DEBUG oslo_concurrency.lockutils [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:34:59.978 27729 DEBUG nova.objects.instance [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:35:00.050 27729 DEBUG nova.policy [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:35:01.058 27729 INFO nova.osapi_compute.wsgi.server [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3860 time: 1.2608840
2018-06-28 09:35:15.124 27733 DEBUG nova.osapi_compute.wsgi.server [req-5344b9c1-28d6-4323-b591-8612ad820165 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] (27733) accepted ('10.0.20.25', 60664) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:35:15.583 27733 INFO nova.osapi_compute.wsgi.server [req-8378c305-d4c6-4f22-9c29-09b2351200c4 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497 HTTP/1.1" status: 404 len: 382 time: 0.4562039
2018-06-28 09:35:15.605 27733 DEBUG nova.api.openstack.wsgi [req-84014c7e-c427-4a64-a51d-e41801e228c9 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7f4f4cccbd10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:35:15.608 27733 INFO nova.osapi_compute.wsgi.server [req-84014c7e-c427-4a64-a51d-e41801e228c9 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0131881
2018-06-28 09:35:16.020 27733 DEBUG nova.api.openstack.wsgi [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:35:16.028 27733 DEBUG nova.compute.api [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:35:16.047 27733 DEBUG oslo_concurrency.lockutils [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:16.048 27733 DEBUG oslo_concurrency.lockutils [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:16.120 27733 DEBUG nova.compute.api [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:35:16.121 27733 DEBUG nova.compute.api [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:35:16.123 27733 DEBUG oslo_concurrency.lockutils [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:16.123 27733 DEBUG oslo_concurrency.lockutils [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:16.225 27733 INFO nova.osapi_compute.wsgi.server [req-d120b159-dbef-4dea-b845-a2e1edd03822 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 786 time: 0.2202611
2018-06-28 09:35:16.256 27733 DEBUG nova.api.openstack.wsgi [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:35:16.271 27733 DEBUG nova.compute.api [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:35:16.297 27733 DEBUG oslo_concurrency.lockutils [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:16.298 27733 DEBUG oslo_concurrency.lockutils [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:16.421 27733 DEBUG oslo_concurrency.lockutils [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:16.422 27733 DEBUG oslo_concurrency.lockutils [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:16.423 27733 DEBUG nova.objects.instance [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:35:16.495 27733 DEBUG nova.policy [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:35:17.448 27733 INFO nova.osapi_compute.wsgi.server [req-bf78266a-bf4d-44e2-93a8-93f1ff44a85a 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3860 time: 1.2052021
2018-06-28 09:35:31.407 27731 DEBUG nova.osapi_compute.wsgi.server [req-ba70d1f1-2b7e-48a0-9964-ff4246d73267 - - - - -] (27731) accepted ('10.0.20.25', 60710) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:35:31.419 27731 WARNING keystonemiddleware.auth_token [req-ba70d1f1-2b7e-48a0-9964-ff4246d73267 - - - - -] Using the in-process token cache is deprecated as of the 4.2.0 release and may be removed in the 5.0.0 release or the 'O' development cycle. The in-process cache causes inconsistent results and high memory usage. When the feature is removed the auth_token middleware will not cache tokens by default which may result in performance issues. It is recommended to use  memcache for the auth_token token cache by setting the memcached_servers option.
2018-06-28 09:35:33.056 27731 INFO nova.osapi_compute.wsgi.server [req-1d869c45-d66d-46b4-afcf-7a671fe26d24 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497 HTTP/1.1" status: 404 len: 382 time: 1.6448200
2018-06-28 09:35:33.081 27731 DEBUG nova.api.openstack.wsgi [req-2bb7095c-3dca-4eae-b438-b653be451fa4 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7f4f4cccbd10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:35:33.084 27731 INFO nova.osapi_compute.wsgi.server [req-2bb7095c-3dca-4eae-b438-b653be451fa4 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0155339
2018-06-28 09:35:33.477 27731 DEBUG nova.api.openstack.wsgi [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:35:33.488 27731 DEBUG nova.compute.api [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:35:33.524 27731 DEBUG oslo_db.sqlalchemy.engines [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:35:33.564 27731 DEBUG oslo_concurrency.lockutils [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:33.566 27731 DEBUG oslo_concurrency.lockutils [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:33.588 27731 DEBUG oslo_db.sqlalchemy.engines [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:35:33.689 27731 DEBUG nova.compute.api [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Found 2 cells: 00000000-0000-0000-0000-000000000000(cell0),207d3421-8e68-4ec6-a61c-47da5c989f5f(default) load_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:236
2018-06-28 09:35:33.690 27731 DEBUG nova.compute.api [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:35:33.691 27731 DEBUG nova.compute.api [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:35:33.693 27731 DEBUG oslo_concurrency.lockutils [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:33.698 27731 DEBUG oslo_concurrency.lockutils [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.005s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:33.718 27731 DEBUG oslo_db.sqlalchemy.engines [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] MySQL server mode set to STRICT_TRANS_TABLES,STRICT_ALL_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,TRADITIONAL,NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION _check_effective_sql_mode /usr/lib/python2.7/site-packages/oslo_db/sqlalchemy/engines.py:285
2018-06-28 09:35:33.824 27731 INFO nova.osapi_compute.wsgi.server [req-5d45c988-f179-4613-936a-6e9ef6ab91ac 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 786 time: 0.3625321
2018-06-28 09:35:33.851 27731 DEBUG nova.api.openstack.wsgi [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:35:33.857 27731 DEBUG nova.compute.api [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:35:33.883 27731 DEBUG oslo_concurrency.lockutils [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:33.884 27731 DEBUG oslo_concurrency.lockutils [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:34.370 27731 DEBUG oslo_concurrency.lockutils [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:35:34.371 27731 DEBUG oslo_concurrency.lockutils [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:35:34.371 27731 DEBUG nova.objects.instance [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:35:34.442 27731 DEBUG nova.policy [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:35:35.448 27731 INFO nova.osapi_compute.wsgi.server [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3860 time: 1.6115992
2018-06-28 09:36:07.140 27732 DEBUG nova.osapi_compute.wsgi.server [req-5e726eeb-4787-468b-98bc-c308270a5bfa 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] (27732) accepted ('10.0.20.25', 33576) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:36:07.397 27732 DEBUG nova.api.openstack.wsgi [req-e49d3884-c960-4ee0-97f0-3064e14fce38 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:07.404 27732 DEBUG nova.compute.api [req-e49d3884-c960-4ee0-97f0-3064e14fce38 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Failed to fetch instance by id tempest-INSTANCE_SAMPLE-1530171027 get /usr/lib/python2.7/site-packages/nova/compute/api.py:2329
2018-06-28 09:36:07.405 27732 INFO nova.api.openstack.wsgi [req-e49d3884-c960-4ee0-97f0-3064e14fce38 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] HTTP exception thrown: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530171027.
2018-06-28 09:36:07.406 27732 DEBUG nova.api.openstack.wsgi [req-e49d3884-c960-4ee0-97f0-3064e14fce38 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Returning 404 to user: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530171027. __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-28 09:36:07.408 27732 INFO nova.osapi_compute.wsgi.server [req-e49d3884-c960-4ee0-97f0-3064e14fce38 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 404 len: 532 time: 0.2619910
2018-06-28 09:36:07.440 27732 DEBUG nova.api.openstack.wsgi [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:07.446 27732 DEBUG nova.compute.api [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:36:07.465 27732 DEBUG oslo_concurrency.lockutils [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:07.467 27732 DEBUG oslo_concurrency.lockutils [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:07.534 27732 DEBUG nova.compute.api [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:36:07.535 27732 DEBUG nova.compute.api [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:36:07.536 27732 DEBUG oslo_concurrency.lockutils [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:07.537 27732 DEBUG oslo_concurrency.lockutils [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:07.616 27732 INFO nova.osapi_compute.wsgi.server [req-a353f212-38f6-4901-9fcc-4ecb0a5041cb 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 784 time: 0.1876290
2018-06-28 09:36:07.641 27732 DEBUG nova.api.openstack.wsgi [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:07.646 27732 DEBUG nova.compute.api [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:36:07.666 27732 DEBUG oslo_concurrency.lockutils [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:07.667 27732 DEBUG oslo_concurrency.lockutils [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:07.785 27732 DEBUG oslo_concurrency.lockutils [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:07.786 27732 DEBUG oslo_concurrency.lockutils [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:07.788 27732 DEBUG nova.objects.instance [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:36:07.866 27732 DEBUG nova.policy [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:36:09.272 27732 INFO nova.osapi_compute.wsgi.server [req-086a28e7-28cb-4fbf-b525-1849e5d4f330 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3475 time: 1.6415009
2018-06-28 09:36:09.307 27732 DEBUG nova.api.openstack.wsgi [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Azione: 'action', metodo chiamata: <function version_select at 0x7f4f4c81bb90>, corpo: {"addFloatingIp": {"address": "172.24.4.13"}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-28 09:36:09.317 27732 DEBUG nova.compute.api [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:36:09.337 27732 DEBUG oslo_concurrency.lockutils [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:09.338 27732 DEBUG oslo_concurrency.lockutils [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:09.461 27732 WARNING nova.api.openstack.compute.floating_ips [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Info cache is InstanceInfoCache(created_at=2018-06-28T07:34:29Z,deleted=False,deleted_at=None,instance_uuid=4db41e07-e0db-4e40-a6c7-909890e9ed3d,network_info=NetworkModel(),updated_at=None) during associate with no nw_info cache
2018-06-28 09:36:09.463 27732 INFO nova.api.openstack.wsgi [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] HTTP exception thrown: Instance network is not ready yet
2018-06-28 09:36:09.465 27732 DEBUG nova.api.openstack.wsgi [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Returning 400 to user: Instance network is not ready yet __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-28 09:36:09.468 27732 INFO nova.osapi_compute.wsgi.server [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "POST /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d/action HTTP/1.1" status: 400 len: 499 time: 0.1762021
2018-06-28 09:36:20.968 27732 DEBUG nova.osapi_compute.wsgi.server [req-d26a9fc0-4c16-4269-9d28-a21dc8098eb1 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27732) accepted ('10.0.20.25', 33632) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:36:21.612 27732 INFO nova.osapi_compute.wsgi.server [req-72f39a4d-ad1e-4cfa-87e1-185ba5961f65 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497 HTTP/1.1" status: 404 len: 382 time: 0.6400349
2018-06-28 09:36:21.632 27732 DEBUG nova.api.openstack.wsgi [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:21.649 27732 DEBUG nova.compute.api [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Searching by: {u'changes-since': datetime.datetime(2018, 6, 28, 7, 31, 21, 337152, tzinfo=<FixedOffset u'+00:00' datetime.timedelta(0)>)} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:36:21.676 27732 DEBUG nova.api.openstack.wsgi [req-42a85554-0014-49f0-898c-5d7fb6f40e30 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method VersionsController.show of <nova.api.openstack.compute.versionsV21.VersionsController object at 0x7f4f4cccbd10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:21.681 27732 INFO nova.osapi_compute.wsgi.server [req-42a85554-0014-49f0-898c-5d7fb6f40e30 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/ HTTP/1.1" status: 200 len: 783 time: 0.0194988
2018-06-28 09:36:21.699 27732 DEBUG oslo_concurrency.lockutils [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:21.703 27732 DEBUG oslo_concurrency.lockutils [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.004s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:21.815 27732 DEBUG nova.compute.api [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:36:21.817 27732 DEBUG nova.compute.api [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:36:21.820 27732 DEBUG oslo_concurrency.lockutils [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:21.821 27732 DEBUG oslo_concurrency.lockutils [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:22.026 27732 DEBUG oslo_concurrency.lockutils [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:22.027 27732 DEBUG oslo_concurrency.lockutils [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:22.050 27732 DEBUG nova.policy [req-62e1b170-4191-4dda-89dc-8bf99fd11037 0508a2373b814911b1d6723f2f75b8cd 910acf3b84b64f358e48f506530f7cb1 - default default] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'0508a2373b814911b1d6723f2f75b8cd', 'roles': [u'ResellerAdmin', u'admin'], 'user_domain_id': u'default', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'910acf3b84b64f358e48f506530f7cb1', 'project_domain_id': u'default'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:36:22.268 27732 DEBUG nova.api.openstack.wsgi [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:22.276 27732 DEBUG nova.compute.api [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:36:22.297 27732 DEBUG oslo_concurrency.lockutils [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:22.298 27732 DEBUG oslo_concurrency.lockutils [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:22.410 27732 DEBUG nova.compute.api [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:36:22.412 27732 DEBUG nova.compute.api [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:36:22.413 27732 DEBUG oslo_concurrency.lockutils [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:22.414 27732 DEBUG oslo_concurrency.lockutils [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:22.631 27732 DEBUG oslo_concurrency.lockutils [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:22.633 27732 DEBUG oslo_concurrency.lockutils [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.003s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:22.658 27732 DEBUG nova.policy [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:36:22.816 27732 INFO nova.osapi_compute.wsgi.server [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/910acf3b84b64f358e48f506530f7cb1/servers/detail?all_tenants=True&changes-since=2018-06-28T07%3A31%3A21.337152%2B00%3A00 HTTP/1.1" status: 200 len: 3478 time: 1.8191030
2018-06-28 09:36:24.011 27732 INFO nova.osapi_compute.wsgi.server [req-16c04075-4fe6-4f2f-8e1f-8ee0722dbb98 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/detail HTTP/1.1" status: 200 len: 3863 time: 1.7575800
2018-06-28 09:36:51.111 27731 DEBUG nova.osapi_compute.wsgi.server [req-bbdc0706-3afc-4c8f-8b28-37a89005d32b 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27731) accepted ('10.0.20.25', 34758) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:36:51.719 27731 DEBUG nova.api.openstack.wsgi [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:36:51.734 27731 DEBUG nova.compute.api [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:36:51.758 27731 DEBUG oslo_concurrency.lockutils [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:51.759 27731 DEBUG oslo_concurrency.lockutils [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:51.871 27731 DEBUG nova.compute.api [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:36:51.873 27731 DEBUG nova.compute.api [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:36:51.875 27731 DEBUG oslo_concurrency.lockutils [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:51.876 27731 DEBUG oslo_concurrency.lockutils [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:52.086 27731 DEBUG oslo_concurrency.lockutils [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:36:52.088 27731 DEBUG oslo_concurrency.lockutils [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.002s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:36:52.112 27731 DEBUG nova.policy [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:36:52.840 27731 INFO nova.osapi_compute.wsgi.server [req-904d2548-7a23-4356-a71e-78169a295235 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/detail HTTP/1.1" status: 200 len: 3478 time: 1.7237751
2018-06-28 09:37:06.940 27729 DEBUG nova.osapi_compute.wsgi.server [req-19f9fc7e-d257-4890-8bce-002b8e46d13f 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27729) accepted ('10.0.20.25', 34778) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:37:07.400 27729 DEBUG nova.api.openstack.wsgi [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:37:07.406 27729 DEBUG nova.compute.api [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:37:07.424 27729 DEBUG oslo_concurrency.lockutils [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:07.425 27729 DEBUG oslo_concurrency.lockutils [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:07.546 27729 DEBUG nova.compute.api [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:37:07.547 27729 DEBUG nova.compute.api [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:37:07.549 27729 DEBUG oslo_concurrency.lockutils [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:07.550 27729 DEBUG oslo_concurrency.lockutils [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:07.714 27729 DEBUG oslo_concurrency.lockutils [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:07.715 27729 DEBUG oslo_concurrency.lockutils [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:07.738 27729 DEBUG nova.policy [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:37:08.452 27729 INFO nova.osapi_compute.wsgi.server [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/detail HTTP/1.1" status: 200 len: 3478 time: 1.5093541
2018-06-28 09:37:22.503 27729 DEBUG nova.osapi_compute.wsgi.server [req-5813cd47-e50b-4e33-9805-8f2bc11555da 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27729) accepted ('10.0.20.25', 34832) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:37:22.997 27729 DEBUG nova.api.openstack.wsgi [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.detail of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8a2d10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:37:23.007 27729 DEBUG nova.compute.api [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:37:23.023 27729 DEBUG oslo_concurrency.lockutils [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:23.024 27729 DEBUG oslo_concurrency.lockutils [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:23.114 27729 DEBUG nova.compute.api [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:37:23.116 27729 DEBUG nova.compute.api [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:37:23.118 27729 DEBUG oslo_concurrency.lockutils [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:23.119 27729 DEBUG oslo_concurrency.lockutils [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:23.273 27729 DEBUG oslo_concurrency.lockutils [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:23.274 27729 DEBUG oslo_concurrency.lockutils [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:23.294 27729 DEBUG nova.policy [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:37:23.884 27729 INFO nova.osapi_compute.wsgi.server [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/detail HTTP/1.1" status: 200 len: 3478 time: 1.3786681
2018-06-28 09:37:33.299 27729 DEBUG nova.osapi_compute.wsgi.server [req-fc2545e3-7e14-48c1-839f-9e9bbeacafe2 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] (27729) accepted ('10.0.20.25', 34852) server /usr/lib/python2.7/site-packages/eventlet/wsgi.py:883
2018-06-28 09:37:33.755 27729 DEBUG nova.api.openstack.wsgi [req-327e317a-d6bc-495e-ada6-5d6e9980afba 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:37:33.763 27729 DEBUG nova.compute.api [req-327e317a-d6bc-495e-ada6-5d6e9980afba 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Failed to fetch instance by id tempest-INSTANCE_SAMPLE-1530171027 get /usr/lib/python2.7/site-packages/nova/compute/api.py:2329
2018-06-28 09:37:33.764 27729 INFO nova.api.openstack.wsgi [req-327e317a-d6bc-495e-ada6-5d6e9980afba 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] HTTP exception thrown: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530171027.
2018-06-28 09:37:33.765 27729 DEBUG nova.api.openstack.wsgi [req-327e317a-d6bc-495e-ada6-5d6e9980afba 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Returning 404 to user: Impossibile trovare l'istanza tempest-INSTANCE_SAMPLE-1530171027. __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-28 09:37:33.767 27729 INFO nova.osapi_compute.wsgi.server [req-327e317a-d6bc-495e-ada6-5d6e9980afba 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 404 len: 532 time: 0.4663620
2018-06-28 09:37:33.797 27729 DEBUG nova.api.openstack.wsgi [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.index of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c8edc10>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:37:33.805 27729 DEBUG nova.compute.api [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Searching by: {'deleted': False, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', u'name': u'tempest-INSTANCE_SAMPLE-1530171027'} get_all /usr/lib/python2.7/site-packages/nova/compute/api.py:2356
2018-06-28 09:37:33.828 27729 DEBUG oslo_concurrency.lockutils [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:33.830 27729 DEBUG oslo_concurrency.lockutils [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "00000000-0000-0000-0000-000000000000" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:33.896 27729 DEBUG nova.compute.api [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Skipping already-collected cell0 list _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2556
2018-06-28 09:37:33.897 27729 DEBUG nova.compute.api [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Listing 1000 instances in cell 207d3421-8e68-4ec6-a61c-47da5c989f5f(default) _get_instances_by_filters_all_cells /usr/lib/python2.7/site-packages/nova/compute/api.py:2559
2018-06-28 09:37:33.899 27729 DEBUG oslo_concurrency.lockutils [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:33.900 27729 DEBUG oslo_concurrency.lockutils [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:33.980 27729 INFO nova.osapi_compute.wsgi.server [req-c66c8bb6-4663-4514-8e31-0cf898bf88f7 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers?name=tempest-INSTANCE_SAMPLE-1530171027 HTTP/1.1" status: 200 len: 784 time: 0.1966898
2018-06-28 09:37:34.010 27729 DEBUG nova.api.openstack.wsgi [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Calling method '<bound method ServersController.show of <nova.api.openstack.compute.servers.ServersController object at 0x7f4f4c84a0d0>>' _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:612
2018-06-28 09:37:34.013 27729 DEBUG nova.compute.api [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:37:34.024 27729 DEBUG oslo_concurrency.lockutils [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:34.024 27729 DEBUG oslo_concurrency.lockutils [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:34.129 27729 DEBUG oslo_concurrency.lockutils [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:34.130 27729 DEBUG oslo_concurrency.lockutils [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:34.130 27729 DEBUG nova.objects.instance [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lazy-loading 'fault' on Instance uuid 4db41e07-e0db-4e40-a6c7-909890e9ed3d obj_load_attr /usr/lib/python2.7/site-packages/nova/objects/instance.py:1049
2018-06-28 09:37:34.181 27729 DEBUG nova.policy [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Policy check for os_compute_api:os-hide-server-addresses failed with credentials {'service_roles': [], 'user_id': u'9e5ee4e333bc4c978c23ee6dbc8bbc5d', 'roles': [u'admin'], 'user_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded', 'service_project_id': None, 'service_user_id': None, 'service_user_domain_id': None, 'service_project_domain_id': None, 'is_admin_project': True, 'is_admin': True, 'project_id': u'51224f7169ad4f1baed59aaad0b51497', 'project_domain_id': u'6e7dd4ac3939445a9992e2fcb576dded'} authorize /usr/lib/python2.7/site-packages/nova/policy.py:168
2018-06-28 09:37:34.663 27729 INFO nova.osapi_compute.wsgi.server [req-8fe1a563-3be5-4252-a0a7-75dfe0a99e40 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "GET /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d HTTP/1.1" status: 200 len: 3475 time: 0.6638811
2018-06-28 09:37:35.371 27729 DEBUG nova.api.openstack.wsgi [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Azione: 'create', metodo chiamata: <bound method VolumeAttachmentController.create of <nova.api.openstack.compute.volumes.VolumeAttachmentController object at 0x7f4f4c6d7890>>, corpo: {"volumeAttachment": {"device": "/dev/vdb", "volumeId": "1dafffcb-b450-4dc2-96f1-e2a585a4a406"}} _process_stack /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:609
2018-06-28 09:37:35.378 27729 DEBUG nova.compute.api [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] [instance: 4db41e07-e0db-4e40-a6c7-909890e9ed3d] Fetching instance by UUID get /usr/lib/python2.7/site-packages/nova/compute/api.py:2324
2018-06-28 09:37:35.389 27729 DEBUG oslo_concurrency.lockutils [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" acquired by "nova.context.get_or_set_cached_cell_and_set_connections" :: waited 0.000s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:270
2018-06-28 09:37:35.390 27729 DEBUG oslo_concurrency.lockutils [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Lock "207d3421-8e68-4ec6-a61c-47da5c989f5f" released by "nova.context.get_or_set_cached_cell_and_set_connections" :: held 0.001s inner /usr/lib/python2.7/site-packages/oslo_concurrency/lockutils.py:282
2018-06-28 09:37:35.482 27729 INFO nova.api.openstack.wsgi [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] HTTP exception thrown: Impossibile 'attach_volume' l'istanza 4db41e07-e0db-4e40-a6c7-909890e9ed3d mentre si trova in vm_state error
2018-06-28 09:37:35.483 27729 DEBUG nova.api.openstack.wsgi [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] Returning 409 to user: Impossibile 'attach_volume' l'istanza 4db41e07-e0db-4e40-a6c7-909890e9ed3d mentre si trova in vm_state error __call__ /usr/lib/python2.7/site-packages/nova/api/openstack/wsgi.py:1029
2018-06-28 09:37:35.486 27729 INFO nova.osapi_compute.wsgi.server [req-53a3c13f-f1b1-4269-9481-0f73388d7cb0 9e5ee4e333bc4c978c23ee6dbc8bbc5d 51224f7169ad4f1baed59aaad0b51497 - 6e7dd4ac3939445a9992e2fcb576dded 6e7dd4ac3939445a9992e2fcb576dded] 10.0.20.25 "POST /v2.1/51224f7169ad4f1baed59aaad0b51497/servers/4db41e07-e0db-4e40-a6c7-909890e9ed3d/os-volume_attachments HTTP/1.1" status: 409 len: 580 time: 0.1270921
